ios/RNCNetInfo.m

```diff
@@ -12,7 +12,8 @@
 #include <arpa/inet.h>
 
 #if !TARGET_OS_TV
-@import CoreTelephony;
+#import<CoreTelephony/CTCarrier.h>
+#import<CoreTelephony/CTTelephonyNetworkInfo.h>
```

Comment:
```suggestion
#import <CoreTelephony/CTCarrier.h>
#import <CoreTelephony/CTTelephonyNetworkInfo.h>
```

Seems reasonable, strange that the `@` import style doesn't work, just on Catalyst.